oneflow/user/kernels/upsample_kernel.h

```diff
@@ -138,7 +138,7 @@ OF_DEVICE_FUNC T cubic_convolution2(const T x, const T A) {
 
 template<typename T>
 OF_DEVICE_FUNC void get_cubic_upsample_coefficients(T coeffs[4], const T t) {
-  T A = -0.75;
+  float A = -0.75;
```

Comment:
**This fix has accually no effect.**
`A` will be passed to the second parameter typed `const T` of `cubic_convolution1<T>` and `cubic_convolution2<T>`, and obviously an implicit cast will be performed due to `T = int`.